pkg/controllers/dynakube/version/reconciler.go

```diff
@@ -56,7 +56,7 @@ func (r *reconciler) ReconcileOneAgent(ctx context.Context, dynakube *dynatracev
 
 func (r *reconciler) ReconcileActiveGate(ctx context.Context, dynakube *dynatracev1beta1.DynaKube) error {
 	updaters := []StatusUpdater{
-		newActiveGateUpdater(dynakube, r.apiReader, r.dtClient, r.registryClient),
+		newActiveGateUpdater(dynakube, r.apiReader, r.dtClient),
 		newSyntheticUpdater(dynakube, r.apiReader, r.dtClient, r.registryClient),
```

Comment:
shouldn't it be also removed from the `SyntheticUpdater` (its just an activegate, so we could use the same endpoint?)

Reply:
Synthetic is a different image (so probably different version), no related `metainfo` endpoint